### ISSUE TITLE
Objective-C selector conflics

### ISSUE THREAD

**Issue by M-Porter** [Ilhasoft/ISScrollViewPageSwift#5]
I got the error message below when trying to use this library:

```
/Users/mporter/dev/here/HereTest/HereTest/ISScrollViewPageOLD.swift:121:10: Method 'setViewControllers' with Objective-C selector 'setViewControllers:' conflicts with setter for 'viewControllers' with the same Objective-C selector
```

The method that causes this error is:

```
func setViewControllers(viewControllers:[UIViewController]){
        self.viewControllers! = viewControllers;
}
```

I was able to fix the error by changing member `viewControllers` to `viewControllersArr` (or anything other than `viewControllers` for that matter)

**Comment by danielblx** [comment]
Ok! 
In the next week i will fix it.

**Comment by danielblx** [comment]
Fix in version 0.0.4
